packages/shared/src/components/post/PostHeaderActions.tsx

```diff
@@ -118,31 +49,14 @@ export function PostHeaderActions({
           </Button>
         </SimpleTooltip>
       )}
-      <SimpleTooltip placement="bottom" content="Options">
```

Comment:
The menu options seemed similar components in both posts so decided to refactor to a new component. All removals here are from the refactoring.